packages/components/psammead-embed-error/src/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { string, bool } from 'prop-types';
+import styled from 'styled-components';
+import { C_CHALK, C_EBON } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING_TRPL, GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { GEL_BODY_COPY } from '@bbc/gel-foundations/typography';
+import { BBC_BLOCKS, coreIcons } from '@bbc/psammead-assets/svgs';
+
+const GEL_SPACING_QRT = '0.125rem';
+
+const StyledEmbedError = styled.div`
+  ${({ service }) => getSansRegular(service)}
+  ${GEL_BODY_COPY};
+  background-color: ${C_CHALK};
+  background-image: url(data:image/svg+xml;base64,${BBC_BLOCKS});
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: 38.2%;
+  color: ${C_EBON};
+  display: flex;
+  flex-direction: column;
+  height: ${({ fillViewport }) => (fillViewport ? '100vh' : '100%')};
+  justify-content: flex-end;
+`;
+
+const StyledErrorMessage = styled.div`
+  display: flex;
+  margin: ${GEL_SPACING_TRPL};
+
+  strong {
+    font-weight: normal;
+  }
+`;
+
+const IconWrapper = styled.span`
+  margin-right: ${GEL_SPACING};
```

Comment:
This reminds me, how will this work for RTL sites?

Reply:
Good spot. I've added support using symmetrical margins:

![Screen Shot 2019-11-18 at 14 03 54](https://user-images.githubusercontent.com/1430787/69060136-c1796600-0a0e-11ea-832f-b16bc714ff6b.png)
![Screen Shot 2019-11-18 at 14 04 12](https://user-images.githubusercontent.com/1430787/69060137-c1796600-0a0e-11ea-97ae-bb093a7abd81.png)